src/zjs_timers.c

```diff
@@ -108,6 +108,23 @@ static bool delete_timer(int32_t id)
     return false;
 }
 
+void zjs_timers_free_timers()
+{
+    for (zjs_timer_t **ptm = &zjs_timers; *ptm; ptm = &(*ptm)->next) {
+        zjs_timer_t *tm = *ptm;
+        int i;
```

Comment:
Move 'i' down to the for loop (for (int i = 0...) since that is where it is only used.